Example/Sources/MockMessage.swift

```diff
@@ -32,37 +32,38 @@ struct MockMessage: MessageType {
 	var sender: Sender
 	var sentDate: Date
 	var data: MessageData
+    var type: String
 	
-    init(data: MessageData, sender: Sender, messageId: String, date: Date) {
+    init(data: MessageData, sender: Sender, messageId: String, date: Date, type: String) {
```

Comment:
@alessioarsuffi I think we're getting closer. Do we really need this property? Can't we just check user defaults and add / remove messages from the `messageTypes` array depending on the keys?

Reply:
This was helpful to use in tests, anyway, I've removed everything, let me know if it okay for you :)